clang/test/CIR/CodeGen/vbase.cpp

```diff
@@ -0,0 +1,19 @@
+// RUN: %clang_cc1 -std=c++20 -triple x86_64-unknown-linux-gnu -fclangir-enable -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s
+
+struct A {
+    int a;
```

Comment:
Indent file to 2 spaces

Reply:
Please only mark resolved after you update the diff, otherwise it's confusing to review.

Reply:
yeah, sorry for not uploading the latest version.